enhancements/sig-architecture/224-event-based-manifestwork/README.md

```diff
@@ -0,0 +1,360 @@
+            "selectiveOrphaningRules": []
+        },
+        // manifestConfigOption represents the configurations of a manifest defined in the workload field.
+        // It is an optional property, if not set, the Update type strategy will be used to all the manifests, and no feedback rules will be honored
+        "manifestConfigs": [{
+            // resourceIdentifier represents the group, resource, name and namespace of a resource.
+            "resourceIdentifier": {},
+            // feedbackRules defines what resource status field should be returned. If it is not set or empty, no feedback rules will be honored.
+            "feedbackRules": [{
+                // Type defines the option of how status can be returned, it can be jsonPaths or wellKnownStatus.  If the type is JSONPaths, the user should specify the jsonPaths field. If the type is WellKnownStatus, certain common fields of status defined by a rule only If these status fields do not exist, no values will be reported.
+                // required
+                "type": "",
+                // JsonPaths defines the json path under the status field to be synced.
+                // optional
+                "jsonPaths": []
+            }],
+            "updateStrategy": {
+                // type defines the strategy to update this manifest, default value is Update.
+                // Update type means to update resources by an update call.
+                // CreateOnly type means do not update resources based on the current manifest.
+                // ServerSideApply type means to update resource 
+                // using server side apply with work-controller as the field manager required.
+                "type": "",
+                // serverSideApply defines the configuration for server side apply. It is honored only when type of updateStrategy is ServerSideApply
+                // optional
+                "serverSideApply": {
+                    // Force represents to force apply the manifest.
+                    // optional
+                    "force": false,
+                    // FieldManager is the manager to apply the  resource. It is work-agent by default, but can be another name with work-agent as the prefix.
+                    // optional
+                    "fieldManager": ""
+                }
+            }
+        }]
+    }
+}
+```
+
+### Status Event
+
+The work agent communicates the state of the resources it manages to the respective source using this message.
+
+Event transfer flow:
+
+1. The source subscribes to the status event topic `/sources/<source-id>/clusters/+/manifestsstatus` from the broker, for example: `/sources/sd3ded4-mwrs-controller/clusters/+/manifestsstatus`.
+2. The work agent publishes the status of the resources using the status event to topic `/sources/<source-id>/clusters/<cluster-id>/manifestsstatus`, for example: `/sources/sd3ded4-mwrs-controller/clusters/cluster1/manifestsstatus`.
```

Comment:
nit: use `/sources/sd3ded4-mwrs-agent-controller/clusters/cluster1/manifestsstatus` instead of `/sources/sd3ded4-mwrs-controller/clusters/cluster1/manifestsstatus`

Reply:
done